crates/api/api_utils/src/notify.rs

```diff
@@ -434,13 +418,13 @@ mod tests {
 
     // Sara replied to timmys comment
     NotifyData {
-      post: &data.timmy_post,
-      comment_opt: Some(&data.sara_comment),
-      creator: &data.sara.person,
-      community: &data.community,
+      post: data.timmy_post.clone(),
+      comment_opt: Some(data.sara_comment.clone()),
+      creator: data.sara.person.clone(),
+      community: data.community.clone(),
```

Comment:
Rather than doing all these clones, I think it does make sense to go back to using references.

Reply:
Cloning is necessary to spawn the background task which processes notifications and sends emails. References are not valid in that case. Otherwise the api response would have to wait until notification code is finished processing and emails are all sent.

Reply:
If you're spawning a background task and need to take ownership but don't want to clone the data, maybe an `Arc` would be good here?

Reply:
That's not necessary here, we're not doing thread-safe sharing.

clone works then.